mayastor/src/lvs/lvs_pool.rs

```diff
@@ -244,6 +257,85 @@ impl Lvs {
         }
     }
 
+    /// imports a pool based on its name, uuid and base bdev name
+    pub async fn import_from_args(args: PoolArgs) -> Result<Lvs, Error> {
```

Comment:
Is there are reason this fn is not called "import(args: poolArgs)"  ?

Reply:
There is another function called "import(name: &str, bdev: &str)". If we change that now then a lot of rust test may break.